.gitignore

```diff
@@ -47,6 +47,7 @@ local.properties
 
 # shell scripts
 *.sh
+!.evergreen/*.sh
```

Comment:
We need to track scripts in the `.evergreen` directory, e.g., the script added in this PR.